sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/WebSocketsProxyConnectionHandler.java

```diff
@@ -190,8 +190,10 @@ public void onTransportError(Event event) {
         final URI url = createURI(fullyQualifiedNamespace, port);
         final InetSocketAddress address = new InetSocketAddress(hostNameParts[0], port);
 
-        logger.atError()
-            .log("Failed to connect to url: '{}', proxy host: '{}'", url, address.getHostString(), ioException);
+        logger.atWarning()
+            .addKeyValue("url", url)
+            .addKeyValue("proxyHost", address.getHostString())
+            .log("Failed to connect.", ioException);
```

Comment:
Our current proxy implementation (extended to proton-extensions) does not signal a useful error, but a general transport-error gets reported in common transport level, so I was wondering this external proxy specific Connect error should be in error-level to detect problem easier. WDYT?

Reply:
I don't have a strong opinion here. 

The motivation to make it a warning:
- We'd still report the error and I'd expect users to set level to warning for 3rd party libs by default.
- It seems to be transient

If based on your experience, users don't enable warnings, I'm happy to change the severity back to error.

Reply:
Thanks for sharing the thoughts. What I observed (when exploring custom proxy auth) was, the proxy connect error from proton-extensions layer gets signaled as generic framing-error in core-amqp, those generic error are retried for ever since wrong proxy config won’t get resolved automatically. Thinking a bit more, **I agree with you on using warn**, since this is not a situation (race/one-off) where missing this data would affect diagnosis, as the unable-to-connect behavior would be persistent.

Our messaging log sample configs (in troubleshooting section) do not have entries to enable proton-extension logs, but if incidents happen for web-socket mode, we can instruct users to enable it.